components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/listeners/GatewayStartupListener.java

```diff
@@ -99,4 +102,40 @@ public void invoke() {
         }
     }
 
-}
+
+    public void startListener() {
+        new Thread(this).start();
+    }
+
+    @Override
+    public void run() {
+        deployArtifactsInGateway();
+    }
+
+    private void deployArtifactsInGateway() {
+
+        long retryDuration =
```

Comment:
I think we can have this declaration in single line.

Reply:
Fixed in the latest version https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/listeners/GatewayStartupListener.java#L164